VideoLocker/src/main/java/org/edx/mobile/module/notification/NotificationPayload.java

```diff
@@ -0,0 +1,122 @@
+package org.edx.mobile.module.notification;
+
+import android.text.TextUtils;
+
+import com.google.gson.annotations.SerializedName;
+
+import org.w3c.dom.Text;
+
+import java.util.List;
+
+/**
+{
+        "title": "Course Announcement",
+        "loc-args": [
+        "VAL video"
+        ],
+        "action-loc-key": "VIEW_BUTTON",
+        "loc-key": "COURSE_ANNOUNCEMENT_NOTIFICATION_BODY",
+        "action": "course.announcement",
+        "title-loc-key": "COURSE_ANNOUNCEMENT_NOTIFICATION_TITLE",
+        "title-loc-args": []
+        }
+ **/
+public class NotificationPayload {
+
+    private @SerializedName("push_hash") String pushHash;
+    private @SerializedName("title") String title;
+    private @SerializedName("loc-args") List<String> locArgs;
+    private @SerializedName("action-loc-key") String actionLocKey;
+    private @SerializedName("loc-key") String localKey;
+    private @SerializedName("action") String action;
+    private @SerializedName("title-loc-key") String titleLocKey;
+    private @SerializedName("title-loc-args") List<String> titleLocArgs;
+    private @SerializedName("course-id") String courseId;
```

Comment:
Eventually, we're going to add different notification types with different payloads so maybe this should be ``CourseUpdateNotificationPayload`` or something like that?

Is there a way to get all the common keys and then have a dictionary of the rest or something?